### PR TITLE
Updated environment, added leafmap, voila and kerchunk. 

### PR DESCRIPTION
**Pull request by betolink**
New packages:
* leafmap: interactive mapping widget
* kerchunk: efficient access to archival formats
* voila: allows to convert a Jupyter Notebook into an interactive dashboard

Updates:
* earthaccess: [0.8.2 -> 0.9.0](https://github.com/nsidc/earthaccess/compare/v0.8.2...v0.9.0)

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -90,14 +91,16 @@ dependencies:
   - nco~=5.1.6
   - pooch==1.6.0
   - zarr~=2.18
+  - kerchunk~=0.2
+  - voila~=0.5
```
Comment (weiji14) [inline environment.yml:95]: Maybe put [voila](https://voila.readthedocs.io/en/stable/index.html) under 'Jupyter tools', or perhaps 'Visualization packages'?

File: environment.yml [hunk 2]
```diff
@@ -76,6 +76,7 @@ dependencies:
   - plotly~=5.18.0
   - seaborn==0.12.1 # statistical plotting with matplotlib
   - geemap~=0.30.0
+  - leafmap~=0.31
```
Comment (weiji14) [inline environment.yml:79]: Trying to sort this section alphabetically, could you move `geemap` and `leafmap` up a few lines? Also, could maybe bump `geemap` to v0.32.0.

**Review by weiji14 — COMMENTED** [review]
Thanks @betolink, just a couple of small suggestions on ordering the packages in the `environment.yml` file.

File: environment.yml (hunk 2 above)
Comment (betolink) [reply environment.yml]: done! @weiji14

**Review by weiji14 — APPROVED** [review]
Awesome, thanks @betolink!

**Pull request merged**